### PR TITLE
Feat/#14 create contact

### PR DESCRIPTION
**Pull request by KhrystynaTkachyk**

### CODE REVIEW

**Comment by mykytapavlov** [comment]
The change looks good, but now, I'm as a user want to extend requirements! :D
I saw somebody [creates phone book](https://github.com/mykytapavlov/python-beginner/issues/86). I want this program also to store as many contacts as I want. Please extend functionality based on what we have in "creates phone book".

**Comment by mykytapavlov** [comment]
more technically, `users` dict should store some reference to `i`

**Comment by KhrystynaTkachyk** [comment]
> more technically, `users` dict should store some reference to `i`

@mykytapavlov Done.

File: entry_tasks/14_create_contact/main.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    users = {}
+    num = int(input("Enter total number of contacts to add: "))
+    for i in range(num):
+        name = input("Enter your name: ")
+        age = int(input("Enter your age: "))
+        address = input("Enter your address: ")
+        phone = input("Enter your phone: ")
+        print('>>> Contact created: ', {f'name: {name}, age: {age}, address: {address}, phone: {phone}'})
+        users[name] = phone
+        #print(f'{name}: {phone}')
```
Comment (mykytapavlov) [inline entry_tasks/14_create_contact/main.py:11]: remove

File: entry_tasks/14_create_contact/main.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    print('Your contact book:')
+    counter = 0
+    record = {}
+    for key, value in users.items():
+        counter += 1
```
Comment (mykytapavlov) [inline entry_tasks/14_create_contact/main.py:16]: Let's assume, loop over users will not print them in the same order as they were inserted (witch is True for the older versions of Python!)
Rework to store `i` somewhere and show it here.

**Review by mykytapavlov — CHANGES_REQUESTED** [review]

**Review by mykytapavlov — APPROVED** [review]

**Pull request merged**